src/components/ComeBackFeedback/ComeBackFeedback.css

```diff
@@ -0,0 +1,56 @@
+.comeback-feedback {
+    width: 100vw;
+    border-top: 1px solid #E5E5E5;
+    text-align: center;
+    margin: 150px auto 0px auto;
+}
+
+img {
+    max-width: 450px;
+}
+
+strong {
+    font-size: 23pt;
+    font-weight: 600;
+}
+
+a {
+    text-decoration: none;
+}
+
+.comeback-container {
+    display: flex;
+    max-width: 1020px;
+    margin: 150px auto 250px auto;
+    justify-content: space-between;
+}
+
+.comeback-text {
+    max-width: 380px;
```

Comment:
maybe consider changing this to rem as well since this will depend on the font size? you can test it out by changing your browser's font size settings while coding